### PR TITLE
Tutorial review for v3

### PR DESCRIPTION
**Pull request by djhi**
I have to change again the two following which are not taken from the tutorial but the simple example

### CODE REVIEW

File: docs/Tutorial.md [hunk 1]
```diff
@@ -645,6 +645,31 @@ const App = () => (
 
 ![Custom home page](./img/dashboard.png)
 
+This dashboard could be used to display aggregated data and react-admin provide some React hooks to make it easier. One of these hooks is `useGetList` and is used like this:
```
Comment (fzaninotto) [inline docs/Tutorial.md]:
```suggestion
This dashboard could be used to display aggregated data and react-admin provides some React hooks to make it easier. One of these hooks is `useGetList` and is used like this:
```

**Comment by fzaninotto** [comment]
The Gifs don't show the mouse pointer anymore...

**Comment by djhi** [comment]
Not sure it's an option with peek, I'll check

**Comment by djhi** [comment]
Looks like its is indeed an option, activated by default but not working...

**Comment by mpaciu** [comment]
Could you upgrade the advanced tutorial of "React Admin Advanced Recipes - Adding Related Records Inline With Custom Forms" to 3.0 version. Maybe this is not the place and you can tell me where. Thanks! Great works!

**Comment by Luwangel** [comment]
> Could you upgrade the advanced tutorial of "React Admin Advanced Recipes - Adding Related Records Inline With Custom Forms" to 3.0 version. Maybe this is not the place and you can tell me where. Thanks! Great works!

Hi, the tutorial has been upgraded a few months ago. https://marmelab.com/blog/2020/04/27/react-admin-tutorials-custom-forms-related-records.html

**Pull request closed without merging**